doc/source/profiling.rst

```diff
@@ -0,0 +1,70 @@
+Profiling Ray
```

Comment:
Does this work? I thought rst requires the underlining to be the same length as the title.